packages/fxa-auth-server/lib/google-maps-services.ts

```diff
@@ -49,7 +50,7 @@ export class GoogleMapsService {
         scope.setContext('googleMapsService', {
           address,
         });
-        Sentry.captureMessage(error.message, Sentry.Severity.Error);
+        Sentry.captureMessage(error.message, 'error' as SeverityLevel);
```

Comment:
Sentry removed the severity enum in the new version?

Reply:
They did! I was also surprised by this. See release notes [here](https://github.com/getsentry/sentry-javascript/blob/develop/MIGRATION.md#severity-severitylevel-and-severitylevels).